src/Obsidian/Cache.ts

```diff
@@ -115,6 +116,21 @@ export function getTasksFromFileContent2(
                 sectionIndex++;
                 tasks.push(task);
             }
+        } else {
+            // 1st:
+            // Root ListItems should not be parents of anything.
+            // This behavior was introduced in DataView plugin, so we want keep it for consistency reasons.
+            // 2nd:
+            // Usually, root listItem has parent = -lineNumber, but for listItems on the top of the file (on 0 line), it is -1.
```

Comment:
Wow, I would never have guessed that -1 is used for list sections starting at the top of the file. And I see it's confirmed by commit da450089abb8e048bc0e6aa7a46a52299efd508e.

Thank you for recording that.